plugins/systemstat.koplugin/main.lua

```diff
@@ -1,6 +1,9 @@
+
```

Comment:
No, each team in MS has its own coding style, but it's not so strict, and easy to be changed.

BOM: https://en.wikipedia.org/wiki/Byte_order_mark.